src/components/common/NftDisplayer/components/ICNSDisplayer/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import Image from '@/commonComponents/Image';
```

Comment:
im thinking about removing `commonComponents` alias, because VSCode autoimports using components/common.

I'll propose to replace with:

`import { Image, Text } from '@/components/common';`

Reply:
Nice, love the idea.